octobot/producers/interface_producer.py

```diff
@@ -16,11 +16,15 @@
 from octobot.channels.octobot_channel import OctoBotChannelProducer
 from octobot.constants import PROJECT_NAME, LONG_VERSION, CONFIG_KEY
 from octobot_backtesting.api.backtesting import is_backtesting_enabled
+from octobot_commons.enums import OctoBotChannelSubjects
 from octobot_services.api.interfaces import create_interface_factory, initialize_global_project_data, is_enabled, \
     start_interfaces, is_enabled_in_backtesting
-from octobot_services.api.notification import create_notifier_factory, is_enabled_in_config
+from octobot_services.api.notification import create_notifier_factory, is_enabled_in_config, \
+    process_pending_notifications
 from octobot_services.interfaces.util.bot import get_bot_api
 from octobot_services.managers.interface_manager import stop_interfaces
+from octobot_services.consumers.octobot_channel_consumer import OctoBotChannelServiceActions as ServiceActions, \
+    OctoBotChannelServiceDataKeys as ServiceKeys
 
 
 class InterfaceProducer(OctoBotChannelProducer):
```

Comment:
As it manages interface and notifier creation it should be named `ServiceProducer`, what do you think ?